src/SmartComponents/RunTaskModal/ConversionTaskInputParameters.js

```diff
@@ -0,0 +1,197 @@
+        isChecked={toBool(elsDisabledValue)}
+        onChange={(_event, value) => {
+          const newValue = value ? 'True' : 'False';
+          setElsDisabledValue(newValue);
+          updateParameters(elsDisabled, newValue);
+        }}
+      />
+      <TextContent>
+        <Text
+          component={TextVariants.p}
+          style={{
+            fontSize: '14px',
+            color: '#6a6e73',
+            paddingLeft: '20px',
+            marginTop: '-20px',
+          }}
+        >
+          If you plan to upgrade to RHEL 8 right after the conversion, you may
+          opt not to use the ELS subscription. Note that the conversion and the
+          subsequent upgrade without an ELS subscription come with a limited
+          support scope per the {convert2RHELSupportLink} and the{' '}
+          {inPlaceSupportLink}.
+        </Text>
+      </TextContent>
+
+      <TextContent>
+        <Text component={TextVariants.h4}>
+          Ignore specific pre-conversion analysis check results.
+        </Text>
+        <div style={{ border: 'thin solid black', padding: '5px 10px' }}>
+          <span style={{ fontWeight: 'bold' }}>
+            <Icon status={'warning'}>
+              <ExclamationTriangleIcon />
+            </Icon>{' '}
+            Resolve issues if possible.
+          </span>
+          <Text component={TextVariants.p}>
+            Use these at your discretion and resolve the issues if possible.
+            Overriding issues can cause the conversion to fail, resulting in a
+            deteriorated system state.
+          </Text>
+        </div>
+      </TextContent>
+
+      <Checkbox
+        id={unavailableKmods.key}
+        label={unavailableKmods.title}
+        description={unavailableKmods.description}
+        isChecked={toBool(unavailableKmodsValue)}
+        onChange={(_event, value) => {
+          const newValue = value ? '1' : '0';
+          setUnavailableKmodsValue(newValue);
+          updateParameters(unavailableKmods, newValue);
+        }}
+      />
+      <Checkbox
+        id={skipKernelCheck.key}
+        label={skipKernelCheck.title}
+        description={skipKernelCheck.description}
+        isChecked={toBool(skipKernelCheckValue)}
+        onChange={(_event, value) => {
+          const newValue = value ? '1' : '0';
+          setSkipKernelCheckValue(newValue);
+          updateParameters(skipKernelCheck, newValue);
+        }}
+      />
+      <Checkbox
+        id={skipPackageCheck.key}
+        label={skipPackageCheck.title}
+        description={skipPackageCheck.description}
+        isChecked={toBool(skipPackageCheckValue)}
+        onChange={(_event, value) => {
+          const newValue = value ? '1' : '0';
+          setSkipPackageCheckValue(newValue);
+          updateParameters(skipPackageCheck, newValue);
+        }}
+      />
+    </Form>
+  );
+};
+
+ConversionTaskInputParameters.propTypes = {
+  slug: propTypes.string,
+  parameters: propTypes.array,
```

Comment:
Since we don't do any optional chaining, it would be great if we could mark it as required with isRequired

Reply:
Done!